From 596ed4a3481735b7d286da2701ada64e60e595cc Mon Sep 17 00:00:00 2001
From: Dmitry Rozhkov <dmitry.rozhkov@linux.intel.com>
Date: Tue, 11 Jul 2017 12:47:41 +0300
Subject: [PATCH] don't use INSTALL_INTERFACE

The build system adds `include` as a system headers folder, but in combination
with having --sysroot option set for gcc makes gcc ignore the folder.

Upstream-Status: Inappropriate [the line is legit, but breaks in Yocto]
diff --git a/src/cpp/CMakeLists.txt b/src/cpp/CMakeLists.txt
index d9cdd9a..f9e6038 100644
--- a/src/cpp/CMakeLists.txt
+++ b/src/cpp/CMakeLists.txt
@@ -83,7 +83,6 @@ else()
     # Define public headers
     target_include_directories(${PROJECT_NAME} PUBLIC
         $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include> $<BUILD_INTERFACE:${PROJECT_BINARY_DIR}/include>
-        $<INSTALL_INTERFACE:include>
         )
 endif()
 
